### PR TITLE
Create _vercel.yean.json

### PR DESCRIPTION
**Pull request by joshex150**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x ] The file is in the `domains` folder and is in the JSON format.
- [x ] The file's name is all lowercased and alphanumeric. 
- [x ] You have completed your website. 
- [x ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/_vercel.yean.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "username": "joshex150",
+      "email": "joshex150@gmail.com"
+    },
+    "record": {
+      "TXT": "vc-domain-verify=yean.is-a.dev,c671dbf711ab7400712a"
```
Comment (DEV-DIBSTER) [inline domains/_vercel.yean.json:7]: Assuming this is for a vercel domain, but I'm not seeing a file with yean.json in the domains folder or this PR, make sure you have that included as well.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/_vercel.yean.json (hunk 1 above)
Comment (joshex150) [reply domains/_vercel.yean.json:7]: I've done this a couple of times now, what exactly should I include in the second file

Comment (DEV-DIBSTER) [reply domains/_vercel.yean.json:7]: Preferably the actual A/CNAME records that Vercel wants. Currently you're just requesting a verification, but that's useless if the site is not being pointed towards Vercel.

Refer to this guide for further details: https://www.is-a.dev/docs/vercel/

Comment (joshex150) [reply domains/_vercel.yean.json:7]: I understand what to do now, thank you!

**Pull request closed without merging**